index.html

```diff
@@ -389,6 +389,18 @@ <h4 class="uppercase mb16">Blog</h4>
 					</div>
 				</div>
 				<div class="row">
+					<div class="col-md-3 col-sm-6">
+						<a href="https://participationgcin.wordpress.com/" target="_blank">
+							<div class="image-tile outer-title text-center">
+								<img alt="Pic" src="img/students/sudhanshuedoc.jpg" height="240" width="240">
```

Comment:
Don't add yourself statically, insert your data to `_data/students.yml`, please see the readme.md for an in depth guide. :)

Reply:
Above u said resize ur image which image should I resize Introduction part one or homepage part of my blog
??????

Reply:
@Sudhanshuedoc pls resize img/students/sudhanshuedoc.jpg, the image added with this PR.

Reply:
Ok sir got u. Sir can I do that tomorrow

Reply:
????

Reply:
Actually my pc stopped working so am replying u with my phone so can I do that tomorrow?????

Reply:
@Sudhanshuedoc take your own time,
no need to call me sir, I am fellow student like you
take your own time, we will be here itself, it's open source! Enjoy

Reply:
Am out with my deadline that will not not gonna affect my project na????

Reply:
???

Reply:
@Sudhanshuedoc you have I guess 7 days time to work on this issue, if you request mentors will give you more time, if you fail also, your task will get rejected. Usually, the extension time will be enough for you. If you can't get it in extension time also, don't worry find a new task to work on, this PR will remain here. You have all the time in the world to make works here, (7 days for task). :)